app.js

```diff
@@ -0,0 +1,95 @@
+
+let  headlines = []
+let summary = []
+
+function renderHeadlines() {
+  htmlstr="";
+  headlines.map((element, index) => {
+    htmlstr += `<a class='headline-box' id='headline${index}'><img id='headline-picture' src='${element.thumbnail}'><div id='headline-text'>${element.webTitle}</div>`
```

Comment:
This line is very long, perhaps can split the HTML section to a separate function